protocol/payloads.proto

```diff
@@ -188,6 +188,7 @@ message ChunkedMedia {
 
 message UserBio {
     string bio = 1;
+    optional uint64 updated_at = 2;
```

Comment:
before checking in can you call this updated_at_epoch_ms to match what we have in protocol.proto?. With dates it's good to be explicit.